example_2.py

```diff
@@ -0,0 +1,25 @@
+operations_count = 0
+
+def main():
+    ask_again = True
+    while(ask_again):
+        a = input("Enter the numerator: ")
+        b = input("Enter the denominator: ")
+        result = perform_division(a,b)
+        print(result)
+        ask_again = input("Do you want to perform another operation? Enter yes or no: ")
+        if(ask_again == 'yes'):
+            ask_again = True
+        else:
+            ask_again = False
+            print("You performed " + str(operations_count) + " operations, bye!")
+
+def perform_division(a,b):
+    global operations_count
+    try:
+        operations_count += 1
+        return int(a)/int(b)
+    except Exception as e:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are generically passing an `Exception` object without performing any other operation on it. This may hide error conditions that can otherwise be quickly detected and addressed. We recommend that you catch a more specific exception. If the code must broadly catch all exceptions, consider logging the stack trace using the [logging.exception()](https://docs.python.org/3/library/logging.html#logging.exception) API. For example,

```
try:
    x = 1 / 0
except ZeroDivisionError as e:
    logging.exception(&#39;ZeroDivisionError: %s&#39;, e)

```